### PR TITLE
feat: apply new design

### PR DESCRIPTION
**Pull request by fpasquet**

### CODE REVIEW

File: .github/actions/restore-npm-cache/action.yml [hunk 1]
```diff
@@ -9,3 +9,6 @@ runs:
         path: "**/node_modules"
         key: cache-node-modules-${{ hashFiles('**/package-lock.json') }}
         restore-keys: cache-node-modules-npm run
+    - name: Prepare project
```
Comment (ch3ric) [inline .github/actions/restore-npm-cache/action.yml:12]: Maybe move to a different workflow file or rename this one because this is not related to 'npm cache'

**Review by ch3ric — COMMENTED** [review]
I see at least 2 'intro' png images that are not transparent and that should be in 'jpg' instead, to be more optimized

File: src/components/Blocks/HomeIntroBlock/HomeIntroBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import React from 'react';
+
+export interface HomeIntroBlockProps {
+  intro: string;
+  title: string;
+  description: string;
+  elevenLabsLink: { label: string } & React.ComponentPropsWithoutRef<'a'>;
+}
+
+//pl={{ xs: 'xs', md: 'xxl-2' }}
```
Comment (ch3ric) [inline src/components/Blocks/HomeIntroBlock/HomeIntroBlock.tsx:13]: Oops

File: src/containers/HeaderContainer/useHeaderContainer.tsx [hunk 3]
```diff
@@ -79,12 +73,34 @@ export const useHeaderContainer = (): HeaderProps => {
...
-    onToggleSearch,
+    categories:
+      layoutTemplateData.categories.map((currentCategoryName) => ({
+        hrefLang: i18n.language,
+        href: generatePath(currentCategoryName === 'all' ? PATHS.HOME : PATHS.CATEGORY, {
+          lang: i18n.language,
+          categoryName: currentCategoryName,
+        }),
+        label: currentCategoryName === 'all' ? t('categories.all') : t(`categories.${currentCategoryName}`),
+        /*isActive: currentCategoryName === categoryName ? true : Boolean(!categoryName && currentCategoryName === 'all'),*/
```
Comment (ch3ric) [inline src/containers/HeaderContainer/useHeaderContainer.tsx:90]: Oops

File: .github/actions/restore-npm-cache/action.yml (hunk 1 above)
Comment (fpasquet) [reply .github/actions/restore-npm-cache/action.yml:12]: This command is executed when we do an npm install, it is a pre-command, but since we are restoring the cache, it is not executed

**Review by RedHotMan — APPROVED** [review]

File: src/helpers/downloadTranslationsHelper.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  for (const lang of LANGUAGES_AVAILABLE) {
+    const translations = await getTranslations(lang);
+    writeFileSync(
+      resolve(process.cwd(), 'src/translations', `${lang}.translations.json`),
+      JSON.stringify(translations, null, 2),
+      'utf-8'
+    );
+  }
+
+  console.log('🦖 The translations have been uploaded');
```
Comment (ch3ric) [inline src/helpers/downloadTranslationsHelper.ts:28]:
```suggestion
  console.log('🦖 The translations have been downloaded');
```

File: src/entry-server.tsx [hunk 5]
```diff
@@ -22,6 +22,13 @@ export const render = async (options: RenderOptions): Promise<string> => {
   const { query } = createStaticHandler(routes, { basename: BASE_URL });
   const context = await query(options.request);
 
+  if (IS_DEBUG) {
+    const isHomePage = new URL(options.request.url).pathname.replace(BASE_URL, '') === '';
+    if (isHomePage) {
+      await options.i18n.changeLanguage(LanguageEnum.FR);
```
Comment (ch3ric) [inline src/entry-server.tsx:28]: why do we need this hack?

File: src/helpers/prerenderHelper/getSitemapEntries.test.ts [hunk 6]
```diff
@@ -32,32 +34,19 @@ describe('getSitemapEntries', () => {
...
         ],
       },
-      {
-        priority: 0,
-        links: [
-          { lang: 'fr', url: '/fr/search/' },
-          { lang: 'en', url: '/en/search/' },
-        ],
-      },
-      { priority: 0, links: [{ lang: 'fr', url: '/404' }] },
```
Comment (ch3ric) [inline src/helpers/prerenderHelper/getSitemapEntries.test.ts:60]: :+1:

File: src/entry-server.tsx (hunk 5 above)
Comment (fpasquet) [reply src/entry-server.tsx:28]: I have to add a condition, because it is only in the case where we have the debug language

Comment (ch3ric) [reply src/entry-server.tsx:28]: And there is a rendering issue with 'debug language's only on the home page?

File: .github/actions/restore-npm-cache/action.yml [hunk 7]
```diff
@@ -9,3 +9,6 @@ runs:
...
+      shell: bash
+      run: npm run prepare
```
Comment (ch3ric) [inline .github/actions/restore-npm-cache/action.yml:14]: to be executed from a manual github actions workflow as well: in later PR

File: src/containers/TransWithHtml/TransWithHtml.tsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  onlyLineBreak?: boolean;
+}
+
+export const TransWithHtml: React.FC<TransWithHtmlProps> = ({ i18nKey, values, onlyLineBreak }) => (
+  <Trans
+    i18nKey={i18nKey}
+    values={values}
+    components={{
+      br: <br />,
+      ...(onlyLineBreak ? {} : { strong: <Text as="strong" />, em: <Text as="em" italic />, a: <Link /> }),
```
Comment (ch3ric) [inline src/containers/TransWithHtml/TransWithHtml.tsx:15]: support ul/ol lists as well: in later PR

**Review by ch3ric — APPROVED** [review]

File: src/helpers/dataLayerHelper.ts [hunk 9]
```diff
@@ -1,166 +1,12 @@
...
-
-export type DataLayerEventAvailable = DataLayerClickEventAvailable | DatalayerContentSearchEvent;
-
-interface ClickEventType {
-  eventName: DataLayerClickEventAvailable['event'];
-  dataAttribute: string;
-  dataAttributeValue?: string;
-}
-
-const clickEvents: ClickEventType[] = [
```
Comment (ch3ric) [inline src/helpers/dataLayerHelper.ts:68]: tracking clicks is not needed anymore?
